src/Exception/EppoException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Eppo\Exception;
+
+use Exception;
+
+/**
+ * Top-level class to ensure we're only catching Eppo exceptions in graceful mode.
+ */
+abstract class EppoException extends Exception
```

Comment:
Happy to pull this exception work out into another PR if you think it needs it

Reply:
I think this is fine